### ISSUE TITLE
Terraform error: Launch stage annotation should be specified at least as BETA for several services

### ISSUE THREAD

**Issue by krazykid** [google/exposure-notifications-verification-server#1781]
### TL;DR
Using Terraform to deploy the environment. Terraform is reporting the error `Error 400: The feature 'sandbox selector' is not supported in the declared launch stage on resource enx-redirect. The launch stage annotation should be specified at least as BETA. Please visit https://cloud.google.com/run/docs/troubleshooting#launch-stage-validation for in-depth troubleshooting documentation.` when updating services 

**Expected behavior**
Terraform would finish without error

**Observed behavior**
Terraform reported:

```
google_cloud_run_service.rotation: Modifying... [id=locations/us-central1/namespaces/ma-verification-qvtby51btd/services/rotation]
google_cloud_run_service.stats-puller: Modifying... [id=locations/us-central1/namespaces/ma-verification-qvtby51btd/services/stats-puller]
google_cloud_run_service.modeler: Modifying... [id=locations/us-central1/namespaces/ma-verification-qvtby51btd/services/modeler]
google_cloud_run_service.enx-redirect: Modifying... [id=locations/us-central1/namespaces/ma-verification-qvtby51btd/services/enx-redirect]
google_cloud_run_service.adminapi: Modifying... [id=locations/us-central1/namespaces/ma-verification-qvtby51btd/services/adminapi]
google_cloud_run_service.cleanup: Modifying... [id=locations/us-central1/namespaces/ma-verification-qvtby51btd/services/cleanup]
google_cloud_run_service.server: Modifying... [id=locations/us-central1/namespaces/ma-verification-qvtby51btd/services/server]
google_cloud_run_service.apiserver: Modifying... [id=locations/us-central1/namespaces/ma-verification-qvtby51btd/services/apiserver]
google_cloud_run_service.appsync: Modifying... [id=locations/us-central1/namespaces/ma-verification-qvtby51btd/services/appsync]

Error: Error updating Service "locations/us-central1/namespaces/ma-verification-qvtby51btd/services/enx-redirect": googleapi: Error 400: The feature 'sandbox selector' is not supported in the declared launch stage on resource enx-redirect. The launch stage annotation should be specified at least as BETA. Please visit https://cloud.google.com/run/docs/troubleshooting#launch-stage-validation for in-depth troubleshooting documentation.



Error: Error updating Service "locations/us-central1/namespaces/ma-verification-qvtby51btd/services/adminapi": googleapi: Error 400: The feature 'sandbox selector' is not supported in the declared launch stage on resource adminapi. The launch stage annotation should be specified at least as BETA. Please visit https://cloud.google.com/run/docs/troubleshooting#launch-stage-validation for in-depth troubleshooting documentation.



Error: Error updating Service "locations/us-central1/namespaces/ma-verification-qvtby51btd/services/cleanup": googleapi: Error 400: The feature 'sandbox selector' is not supported in the declared launch stage on resource cleanup. The launch stage annotation should be specified at least as BETA. Please visit https://cloud.google.com/run/docs/troubleshooting#launch-stage-validation for in-depth troubleshooting documentation.



Error: Error updating Service "locations/us-central1/namespaces/ma-verification-qvtby51btd/services/server": googleapi: Error 400: The feature 'sandbox selector' is not supported in the declared launch stage on resource server. The launch stage annotation should be specified at least as BETA. Please visit https://cloud.google.com/run/docs/troubleshooting#launch-stage-validation for in-depth troubleshooting documentation.



Error: Error updating Service "locations/us-central1/namespaces/ma-verification-qvtby51btd/services/modeler": googleapi: Error 400: The feature 'sandbox selector' is not supported in the declared launch stage on resource modeler. The launch stage annotation should be specified at least as BETA. Please visit https://cloud.google.com/run/docs/troubleshooting#launch-stage-validation for in-depth troubleshooting documentation.



Error: Error updating Service "locations/us-central1/namespaces/ma-verification-qvtby51btd/services/apiserver": googleapi: Error 400: The feature 'sandbox selector' is not supported in the declared launch stage on resource apiserver. The launch stage annotation should be specified at least as BETA. Please visit https://cloud.google.com/run/docs/troubleshooting#launch-stage-validation for in-depth troubleshooting documentation.



Error: Error updating Service "locations/us-central1/namespaces/ma-verification-qvtby51btd/services/rotation": googleapi: Error 400: The feature 'sandbox selector' is not supported in the declared launch stage on resource rotation. The launch stage annotation should be specified at least as BETA. Please visit https://cloud.google.com/run/docs/troubleshooting#launch-stage-validation for in-depth troubleshooting documentation.



Error: Error updating Service "locations/us-central1/namespaces/ma-verification-qvtby51btd/services/stats-puller": googleapi: Error 400: The feature 'sandbox selector' is not supported in the declared launch stage on resource stats-puller. The launch stage annotation should be specified at least as BETA. Please visit https://cloud.google.com/run/docs/troubleshooting#launch-stage-validation for in-depth troubleshooting documentation.



Error: Error updating Service "locations/us-central1/namespaces/ma-verification-qvtby51btd/services/appsync": googleapi: Error 400: The feature 'sandbox selector' is not supported in the declared launch stage on resource appsync. The launch stage annotation should be specified at least as BETA. Please visit https://cloud.google.com/run/docs/troubleshooting#launch-stage-validation for in-depth troubleshooting documentation.
```

### Reproduction
1. I ran `terraform apply`
2. Terraform did its thing, and erred 

**Environment**
This was in a MacOS shell, trying to create services in GCP

**Additional information**
I was using Terraform to create an EN Verification Server in GCP.

**Comment by mariliamelo** [comment]
This was fixed with #1791
